crates/re_viewport/src/context_menu/actions/add_entities_to_new_space_view.rs

```diff
@@ -149,15 +149,20 @@ fn create_space_view_for_selected_entities(
 
     let mut filter = EntityPathFilter::default();
 
-    for path in entities_of_interest {
-        filter.add_rule(RuleEffect::Include, EntityPathRule::including_subtree(path));
-    }
-
     let target_container_id = ctx
         .clicked_item_enclosing_container_id_and_position()
         .map(|(id, _)| id);
 
-    let space_view = SpaceViewBlueprint::new(identifier, &origin, filter);
+    // TODO(jleibs): Take the `$origin` into account here
```

Comment:
take it into account how? Generate shorter rules with (backwards-substituted) `$origin` in them?

Reply:
Yeah, though after thinking about this more I think it's the wrong thing to do in most cases.  If you included a specific entity, and then you change the origin it's nice if that entity reference is still the same (rather than now being an incorrect relative reference).